### PR TITLE
Fix some typos (found by codespell)

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

File: Formula/bnfc.rb [hunk 1]
```diff
@@ -126,7 +126,7 @@ def install
       space = " "
       check_out_j = <<~EOS
 
-        Parse Succesful!
+        Parse Successful!
```
Comment (fxcoudert) [inline Formula/bnfc.rb:129]: This should not be changed, we're matching the tool's output (typo included)

**Review by fxcoudert — CHANGES_REQUESTED** [review]
Please make it 1 commit per formula, with commit message formatted like this: `avra: fix typo`

File: Formula/bnfc.rb (hunk 1 above)
Comment (stweil) [reply Formula/bnfc.rb]: Done, thank you for your review.

**Comment by fxcoudert** [comment]
Thanks!

**Comment by babariviere** [comment]
@carlocab the CI is marked as `CI-syntax-only`, so tests jobs won't be run

**Comment by carlocab** [comment]
Aha, I didn't catch that. Thanks.

**Comment by fxcoudert** [comment]
Reverted for `luajit` in b822aca874f3a39203fc7ec3c412399882feded9
Apologies, I thought I have reviewed it in full, and I missed that one

**Comment by stweil** [comment]
I am sorry if my PR has broken something, but how could a typo fix in a comment break `luajit`? That sounds strange.

**Pull request merged**